test/connection_test.exs

```diff
@@ -2,7 +2,7 @@ defmodule Janus.ConnectionTest do
   use ExUnit.Case, async: true
   alias Janus.Connection
   alias Janus.Connection.Transaction
-  alias Janus.MockTransport
+  alias Janus.Mock.Transport, as: MockTransport
```

Comment:
Some tests do this, others alias `Mock` and use `Mock.Transport`. Make it consistent either one way or another